source/train/common.py

```diff
@@ -1,8 +1,9 @@
 import warnings
 import numpy as np
+from deepmd.env import tf
 
 data_requirement = {}
-
+activation_fn_dict = {'tf.nn.relu':tf.nn.relu,'tf.nn.relu6':tf.nn.relu6,"tf.nn.softplus":tf.nn.softplus,"tf.sigmoid":tf.sigmoid,"tf.tanh":tf.tanh,"tf.nn.tanh":tf.nn.tanh}
```

Comment:
I'd prefer a simple activation function name:
```suggestion
activation_fn_dict = {
    "relu": tf.nn.relu,
    "relu6": tf.nn.relu6,
    "softplus": tf.nn.softplus,
    "sigmoid": tf.sigmoid,
    "tanh": tf.nn.tanh
}
```

Reply:
That's a nice suggestion. I will change it.